### PR TITLE
This is called ArrayListClass on Java Mode

### PR DESCRIPTION
**Pull request by villares**

### CODE REVIEW

File: mode/examples/Topics/AdvancedData/A_List/Ball.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            self.speed = self.speed * -0.8
+            self.y = height
+
+    def finished(self):
+        # Balls fade out
+        self.life -= 1
+        if self.life < 0:
+            return True
+        else:
+            return False
```
Comment (jdf) [inline mode/examples/Topics/AdvancedData/A_List/Ball.py]: ```return self.life < 0```

File: mode/examples/Topics/AdvancedData/A_List/Ball.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    def display(self):
+        # Dispaly the circle
+        fill(0, self.life)
+        # strike(0, self.life)
```
Comment (jdf) [inline mode/examples/Topics/AdvancedData/A_List/Ball.py]: ?

File: mode/examples/Topics/AdvancedData/A_List/Ball.py (hunk 1 above)
Comment (villares) [inline mode/examples/Topics/AdvancedData/A_List/Ball.py]: <3
I'll do it!

File: mode/examples/Topics/AdvancedData/A_List/Ball.py (hunk 2 above)
Comment (villares) [inline mode/examples/Topics/AdvancedData/A_List/Ball.py]: There is a comment like this on Shiffman's example... may I remove it then?

Comment (jdf) [inline mode/examples/Topics/AdvancedData/A_List/Ball.py]: Are you sure it doesn't say "stroke"?

Comment (villares) [inline mode/examples/Topics/AdvancedData/A_List/Ball.py]: Ugh!!! Sorry!

Comment (villares) [inline mode/examples/Topics/AdvancedData/A_List/Ball.py]: Done. Thank you so much, @jdf

**Pull request merged**